automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -31,48 +29,58 @@ import spray.json._
 import java.util.UUID
 import scala.concurrent.{Await, Future}
 import scala.language.postfixOps
+import org.broadinstitute.dsde.test.pipeline._
 
 @WorkspacesAzureTest
-class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with CleanUp {
-  val owner: Credentials = UserPool.userConfig.Owners.getUserCredential("hermione")
-  val nonOwner: Credentials = UserPool.chooseStudent
-
-  private val azureManagedAppCoordinates = AzureManagedAppCoordinates(
-    UUID.fromString("fad90753-2022-4456-9b0a-c7e5b934e408"),
-    UUID.fromString("f557c728-871d-408c-a28b-eb6b2141a087"),
-    "staticTestingMrg",
-    Some(UUID.fromString("f41c1a97-179b-4a18-9615-5214d79ba600"))
-  )
+class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with BeforeAndAfterAll with CleanUp {
```

Comment:
I am having trouble running this test locally. I don't know why I named the test class differently from the file… can we rename the class as WorkspacesAzureApiSpec so it matches the filename?

Reply:
Updated class name to `WorkspacesAzureApiSpec`